src/Xamarin.Android.Build.Tasks/Utilities/JniRemappingAssemblyGenerator.cs

```diff
@@ -203,8 +203,10 @@ public JniRemappingAssemblyGenerator (TaskLoggingHelper log)
 		public JniRemappingAssemblyGenerator (TaskLoggingHelper log, List<JniRemappingTypeReplacement> typeReplacements, List<JniRemappingMethodReplacement> methodReplacements)
 			: base (log)
 		{
-			this.typeReplacementsInput = typeReplacements ?? throw new ArgumentNullException (nameof (typeReplacements));
-			this.methodReplacementsInput = methodReplacements ?? throw new ArgumentNullException (nameof (methodReplacements));
+			ArgumentNullException.ThrowIfNull (typeReplacements);
+			ArgumentNullException.ThrowIfNull (methodReplacements);
```

Comment:
Undo this change `ArgumentNullException.ThrowIfNull()` does not exist in `netstandard2.0`.